test/Forwarder.test.ts

```diff
@@ -123,7 +123,7 @@ describe("Forwarder", function () {
   });
 });
 
-describe("safeForwardIfNoCodeAt", function () {
+describe("safeForwardCallIfNoCodeAt", function () {
```

Comment:
I noticed that the name was outdated:

```suggestion
describe("callIfContractExists", function () {
```

Also, the name of this function is actually incorrect as it should be `callIfNoContractExists`. Would you mind renaming in this PR?